src/app/data/project-data.ts

```diff
@@ -180,27 +180,29 @@ export const projectsData = [
     },
     {
         name: "ACNH.Guide",
-        description: "User-friendly display of data for bugs, fish, villagers, items, and more",
-        languages: ["en"],
+        description: "A Free Multi-Purpose Guide and Cataloging app with Catalog Scanner integration for New Horizons.",
```

Comment:
Could we use sentence casing here? Just for consistency with most other listings.